examples/tutorials/español/Parte 02 - Intro al Aprendizaje Federado.ipynb

```diff
@@ -10,6 +10,9 @@
     "\n",
     "Autores:\n",
     "- Andrew Trask - Twitter: [@iamtrask](https://twitter.com/iamtrask)\n",
+    "Traductores:\n",
+    "- Arturo Márquez Flores - Twitter [@arturomf94](https://twitter.com/arturomf94)\n",
+    "\n",
     "\n",
     "Traductores:\n",
     "- Arturo Márquez Flores - Twitter: [@arturomf94](https://twitter.com/arturomf94)\n",
```

Comment:
"Autores:\n",
    "- Andrew Trask - Twitter: [@iamtrask](https://twitter.com/iamtrask)\n"


    "Traductores:\n",
    "- Arturo Márquez Flores - Twitter [@arturomf94](https://twitter.com/arturomf94)\n",
    "- Ricardo Pretelt - Twitter: [@ricardopretelt](https://twitter.com/ricardopretelt)"

Reply:
Hi @ricardopretelt, I think that's  an outdated file. Our names are already in the latest version :)

Reply:
ok, my mistake